Solopov/DZ5_AjaxAsynchronousMethodAndCSS/ServletTestList/src/main/java/services/UserService.java

```diff
@@ -0,0 +1,41 @@
+package services;
+
+import entity.User;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class UserService {
+private static Map<String,String> account =new LinkedHashMap<>();//All registered users. key -login, value -password
+    private User currentUser;
+
+    public boolean addAcc(String login, String password){
```

Comment:
I already mentioned - Account is not too long word, to reduce it to Acc:)

Reply:
Sorry, did not notice :) Already changed